src/nerve/frontends/tui/commander/commander.py

```
@@ -104,6 +117,56 @@ def __post_init__(self) -> None:
+        pending_count = sum(1 for b in self.timeline.blocks if b.status == "pending")
+        waiting_count = sum(1 for b in self.timeline.blocks if b.status == "waiting")
+
+        parts.append(f"Blocks: {total_blocks}")
+
+        if pending_count > 0:
+            parts.append(f"⏳ {pending_count}")
+        if waiting_count > 0:
+            parts.append(f"⏸️  {waiting_count}")
+
+        # Clock
+        current_time = datetime.now().strftime("%H:%M:%S")
```

Ruff (0.14.10)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)